src/recipients/RecipientPicker.tsx

```diff
@@ -36,6 +36,7 @@ import colors from 'src/styles/colors'
 import fontStyles from 'src/styles/fonts'
 import Logger from 'src/utils/Logger'
 import networkConfig from 'src/web3/networkConfig'
+import AwesomeDebouncePromise from 'awesome-debounce-promise'
```

Comment:
`lodash.debounce` has weird behavior with promises / requires annoying plumbing: https://github.com/lodash/lodash/issues/4400

Selected `awesome-debounce-promise` based on recommendation from `react-async-hook` documentation.

Reply:
Had you considered using [use-debounce](https://www.npmjs.com/package/use-debounce)? If we're adding a debounce library I'd strongly prefer that as it's more react specific and better maintained. I see the last update of [awesome-debounce-promise](https://www.npmjs.com/package/awesome-debounce-promise) was 4 years ago vs 14 days for [use-debounce](https://www.npmjs.com/package/use-debounce). There was also a fair amount of discussion around why we shouldn't add a debounce library in https://github.com/valora-inc/wallet/pull/3602#discussion_r1157001660.

Reply:
That library has the behavior I'm trying to avoid with `awesome-debounce-promise`.  I can shoehorn an implementation into lodash.debounce for now.